src/routes/docs/products/functions/executions/+page.markdoc

```diff
@@ -50,6 +50,9 @@ Each execution can have one of the follow status.
 - `processing`
 - The function execution has begun and has not finished.
 ---
+- `scheduled`
```

Comment:
When thinking about the lifecycle of an execution, what comes first, scheduled or processing?